cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -2001,4 +2001,13 @@ public static final class MetadataConsumer {
     public static final String METADATA_CONSUMER_EXTENSIONS_ENABLED_LIST = "metadata.consumer.extensions.enabled.list";
     public static final String METADATA_CONSUMER_EXTENSIONS_DIR = "metadata.consumer.extensions.dir";
   }
+
+  /**
+   * Constants for Source Control Management
+   */
+  public static final class SourceControlManagement {
+    public static final String GIT_COMMAND_TIMEOUT_SECONDS = "source.control.git.command.timeout.seconds";
```

Comment:
nit: should this be milliseconds ? Seeing most fields in app-fabric use milliseconds as unit (for consistency)?

Reply:
jGit has a [setTimeout](https://archive.eclipse.org/jgit/docs/jgit-2.3.1.201302201838-r/apidocs/org/eclipse/jgit/api/TransportCommand.html#setTimeout(int)) method that we pass this value to. The method accepts time in seconds, so I've used the same here.